packages/database/src/migrations/20210511042757-MoveLESMISVisualsToNewDashboards-modifies-data.js

```diff
@@ -0,0 +1,177 @@
+'use strict';
```

Comment:
Haven't reviewed the SQL in here because it's fairly dense. In general, we prefer to pull the data out, manipulate it in javascript (which most of us are more familiar/comfortable with), and then put it back in. Because migrations are one-time, the performance isn't such an issue.

Reply:
Still haven't reviewed this haha, will review after the other changes as some of them imply some migration updates